runtime/native_test.go

```diff
@@ -413,6 +413,33 @@ func TestNativeTypeName(t *testing.T) {
 	}
 }
 
+func TestNewNativeField_ChecksInstanceType(t *testing.T) {
```

Comment:
Please remove the underscore in the test name. It's a bit bulky but I don't think underscores have yet been used elsewhere in test names.